References/DeveloperServices.md

```diff
@@ -0,0 +1,51 @@
+# Developer Services
+
+This page is designed to show you some common tools that you might not have been aware of that will improve your development practices and make the life of a developer at ESDC easier.
+
+## Department Built Tools
+
+### Definitive Media Library (DML)
+
+The DML is a list managed by the ITAM Control Board Working Group and contains a list of COTS products that have been permitted to be used within in the department.
+
+[Link Internal to ESDC](http://dialogue/grp/ITAM-GBTI-SD-DL/Site/Active%20Products.aspx)
+
+### Enterprise Web Services
+
+There are a few web services available. They are all supported and documented on [GCcode here](https://gccode.ssc-spc.gc.ca/iitb-dgiit/sds/ews).
+
+### Wet Templates
+
+The CDTS currently has an API to implement the WET 4.0 into any application. You can read more on that on the GCPedia page: [Centrally_Deployed_Templates_Solution_(CDTS)](https://www.gcpedia.gc.ca/wiki/Centrally_Deployed_Templates_Solution_(CDTS)).  
+SDS also offers a more integrated template for .NET MVC, .NET Web Forms and Java applications. The details on that can be found in GCCode page: [GoC Web Templates project](https://gccode.ssc-spc.gc.ca/GOCWebTemplates).
+
+### Web Analytics
+
+The department has a site to view the website analytics on a number of different web sites and applications. <http://web-stats.prv/Index-e.asp> can be used to view the stats of the applications you develop.
+
+### Departmental Service Bus
+
+Details to come...
+
+## 3rd Party Tools
+
+### GC Code
+
+[GC Code](https://gccode.ssc-spc.gc.ca/)
+is using GitLab; you can store your code base here, manage branches and checkins, track bug and tasks, store all of your documentation, all in one tool! SDS is moving to it for all of the EWS and the WET .NET & Java Templates.
+There is a recommendation on GC Code for IITB found here: <http://www.gcpedia.gc.ca/wiki/ESDC_Development_Community_GC_Code_Recommendation>
+
+### NuGet
+
+The department has access to the NuGet repositories as well as hosting its own internal source for home brewed applications.
+Check out the [ESDC NuGet User Guide](https://www.gcpedia.gc.ca/wiki/EDC_NuGet_User_Guide)
```

Comment:
This doc is getting migrated to GitHub, we'll need to make sure the link gets updated when it does, if it hasn't already.